data/overview.yaml

```diff
@@ -119,66 +119,66 @@ Groups:
 Rings:
   ring:
   Unitary ring:
-  ring morphisms:
-  subrings:
+  ring morphisms: 'algebra/ring.html#ring_hom'
+  subrings: 'ring_theory/subring.html#top'
   The ring Z on the integers:
   Product of rings:
-  Ideal of a commutative ring:
+  Ideal of a commutative ring: 'algebra/module.html#ideal'
   quotient rings:
   prime ideals:
-  maximal ideals:
+  maximal ideals: 'ring_theory/ideals.html#ideal.is_maximal'
   Chinese remainder theorem:
   Algebra:
     algebra over a commutative ring:
     associative algebra over a commutative ring:
   Divisibility in commutative integral rings:
-  Irreducible elements:
-  invertible elements:
+  Irreducible elements: 'algebra/associated.html#irreducible' 
+  Invertible elements: 'algebra/invertible.html#invertible'
```

Comment:
Do we want to support multiple links per item? If yes, then we also have `units` and `is_unit`.

Reply:
We can support it, I don't know if we want to support it and I don't know how it would looks like, the syntax is easy though: `[link1, link2, link3]`, if we want to go further we can just do:
```yaml
invertible elements:
    units: …
    is_unit: …
```
It's already supported, but that sounds highly fine grained.